docs/development/bump-k8s.md

```
@@ -0,0 +1,49 @@
+# Bump Kubernetes Dependencies
+
+## Steps
+
+1. **Determine the latest patch release** for the target minor version. K8s Go module versions use the `v0.<minor>.<patch>` scheme (e.g. K8s 1.36 → `v0.36.x`). Run:
+   ```
+   go list -m -versions k8s.io/api | tr ' ' '\n' | grep 'v0.<minor>\.' | tail -1
+   ```
+   Use the latest patch version for all k8s.io modules below.
+
+2. **Bump all k8s.io dependencies** in `go.mod` (both direct and indirect):
+   ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)